.buildkite/scripts/steps/artifacts/build.sh

```diff
@@ -4,8 +4,16 @@ set -euo pipefail
 
 .buildkite/scripts/bootstrap.sh
 
+if [[ "${RELEASE_BUILD:-}" == "true" ]]; then
+  VERSION="$(jq -r '.version' package.json)"
+  RELEASE_ARG="--release"
+else
+  VERSION="$(jq -r '.version' package.json)-SNAPSHOT"
+  RELEASE_ARG=""
+fi
+
 echo "--- Build Kibana Distribution"
-node scripts/build --all-platforms --debug --docker-cross-compile --skip-docker-cloud
+node scripts/build "$RELEASE_ARG --all-platforms --debug --docker-cross-compile --skip-docker-cloud"
```

Comment:
```suggestion
node scripts/build "$RELEASE_ARG" --all-platforms --debug --docker-cross-compile --skip-docker-cloud
```

I think if you wrap the whole thing in quotes, it will be passed as the first argument, instead of as a series of arguments.